php-checks/src/main/java/org/sonar/php/checks/InvertedBooleanCheck.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.php.checks;
+
+import org.sonar.check.Rule;
+import org.sonar.plugins.php.api.tree.Tree.Kind;
+import org.sonar.plugins.php.api.tree.expression.BinaryExpressionTree;
+import org.sonar.plugins.php.api.tree.expression.ParenthesisedExpressionTree;
+import org.sonar.plugins.php.api.tree.expression.UnaryExpressionTree;
+import org.sonar.plugins.php.api.visitors.PHPVisitorCheck;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Rule(key = InvertedBooleanCheck.KEY)
+public class InvertedBooleanCheck extends PHPVisitorCheck {
+
+  public static final String KEY = "S1940";
```

Comment:
I know that we have quite some rules which define such `KEY` constant but I don't know why. It's simpler to just pass the key as a literal to the `@Rule` annotation.
```
@Rule(key = "S1940")
```

Reply:
Okay, but shouldn't we agree on a uniform format?